exporters/tokens/src/generators/__tests__/contentGenerator.test.ts

```diff
@@ -38,7 +38,7 @@ describe('contentGenerator', () => {
         sortByNumValue: false,
       };
 
-      const fileContent = generateFileContent(tokens, mappedTokens, tokenGroups, fileData, false);
+      const fileContent = generateFileContent(tokens, mappedTokens, tokenGroups, fileData, false, undefined, undefined);
```

Comment:
Did you consider using named arguments or a configuration object? It would help with readability of the `false`/`undefined`/similar values.